apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -47,7 +47,7 @@ const Input = <T extends FieldValues>({
 
 	return (
 		<label className={styles["input-label"]}>
-			<span className={styles["input-label-text"]}>{label}</span>
+			{label && <span className={styles["input-label-text"]}>{label}</span>}
```

Comment:
As I mentioned above label must be present on the page anyway -- look A2 of quality criteria. Look how `Select` component implemented and use the same aproach

Reply:
Agree, take a look at the approach with IconButton

Reply:
changed